app/assets/javascripts/pageflow/ui/views/inputs/text_area_input_view.js

```diff
@@ -20,13 +20,17 @@ pageflow.TextAreaInputView = Backbone.Marionette.ItemView.extend({
 
     this.editor = new wysihtml5.Editor(this.ui.textarea[0], {
       toolbar: this.ui.toolbar[0],
+      autoLink: this.options.disableLinks ? 0 : 1,
       parserRules: {
         tags: {
-          u: {},
-          b: {},
-          i: {},
+          em: {rename_tag: "i"},
+          strong: {rename_tag: "b"},
```

Comment:
Identifier 'rename_tag' is not in camel case.<br>Mixed double and single quotes.